azurerm/internal/services/storage/storage_account_customer_managed_key_resource.go

```diff
@@ -62,6 +63,12 @@ func resourceStorageAccountCustomerManagedKey() *pluginsdk.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
+
+			"user_assigned_identity": {
```

Comment:
could we be consistent with the rest of the provider?
```suggestion
			"user_assigned_identity_id": {
```